public/views/walletHome.html

```diff
@@ -412,6 +412,33 @@ <h4 class="title m0">
               </div>
             </div>
 
+            <div class="m20b" ng-init="home.hideAdvSend=true">
+              <a class="button outline light-gray expand tiny m0" ng-click="home.hideAdvSend=!home.hideAdvSend">
+                <i class="fi-widget m3r"></i>
+                <span translate ng-hide="!home.hideAdvSend">Show Advanced options</span>
+                <span translate ng-hide="home.hideAdvSend">Hide Advanced options</span>
+                <i ng-if="home.hideAdvSend" class="icon-arrow-down4"></i>
+                <i ng-if="!home.hideAdvSend" class="icon-arrow-up4"></i>
+              </a>
+              <div class="m10t" ng-hide="home.hideAdvSend">
+                <h4 class="title m0" translate>Fee policy for this transaction</h4>
+                <ul class="no-bullet m0 size-14">
+                  <li ng-repeat="fee in (index.network == 'livenet' ? index.feeLevels.livenet : index.feeLevels.testnet)"
+                    ng-click="home.currentSendFeeLevel = fee.level" class="line-b p20">
+                    {{index.feeOpts[fee.level]|translate}}
+                    <i class="fi-check size-16 right" 
+                      ng-show="(home.currentSendFeeLevel || index.currentFeeLevel) == fee.level"></i>
+                  </li>
+                </ul>
+                <h4 class="title m0" translate>Use unconfirmed funds for this transaction</h4>
+                <div class="p20 line-b">
+                  <span class="size-14" translate>Use Unconfirmed Funds</span>
```

Comment:
One last comment, could you please not capitalize every word. I think "Use unconfirmed funds" is easier to read.